index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fitno">
+    <title>D4niel J Portofolio</title>    
+    <link rel="stylesheet" type="text/css" href="css/normalize.css" media="screen"/>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,200;1,300;1,400;1,500;1,600;1,700;1,800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="css/styles.css" media="screen"/>
+    <script src="https://kit.fontawesome.com/d5e3631d2a.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <nav>
+      <div id="nav-container">
+        <div id="logo">
+          D4NIEL J
+        </div>
+        <div id="menu">
```

Comment:
- [ ] Instead of `div`, it should be `nav`. To wrap menus we will use a `nav` tag as it will make code more readable.